client/payments/payment-recommendations.tsx

```diff
@@ -24,64 +22,23 @@ import { getAdminLink } from '@woocommerce/wc-admin-settings';
  * Internal dependencies
  */
 import './payment-recommendations.scss';
-import { getCountryCode } from '../dashboard/utils';
 import { createNoticesFromResponse } from '../lib/notices';
-import { isWCPaySupported } from '~/tasks/fills/PaymentGatewaySuggestions/components/WCPay';
 
 const SEE_MORE_LINK =
 	'https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/?utm_source=payments_recommendations';
-const DISMISS_OPTION = 'woocommerce_setting_payments_recommendations_hidden';
-type SettingsSelector = WPDataSelectors & {
-	getSettings: (
-		type: string
-	) => { general: { woocommerce_default_country?: string } };
-};
-
-type OptionsSelector = WPDataSelectors & {
-	getOption: ( option: string ) => boolean | string;
-};
 
 export function getPaymentRecommendationData(
 	select: WCDataSelector
 ): {
-	displayable: boolean;
 	recommendedPlugins?: Plugin[];
 	isLoading: boolean;
 } {
-	const { getOption, isResolving: isResolvingOption } = select(
-		OPTIONS_STORE_NAME
-	) as OptionsSelector;
-	const { getSettings } = select( SETTINGS_STORE_NAME ) as SettingsSelector;
 	const { getRecommendedPlugins } = select( PLUGINS_STORE_NAME );
-	const { general: settings } = getSettings( 'general' );
 
-	const hidden = getOption( DISMISS_OPTION );
-	const countryCode =
-		settings && settings.woocommerce_default_country
-			? getCountryCode( settings.woocommerce_default_country )
-			: null;
-	const countrySupported = countryCode
-		? isWCPaySupported( countryCode )
-		: false;
-	const isRequestingOptions = isResolvingOption( 'getOption', [
-		DISMISS_OPTION,
-	] );
```

Comment:
We do not need the dismiss option anymore and don't need the country info either. All of this is handled on the back-end now, so we only do one request.